lib/core/src/server/config/styleWebpackRules.js

```diff
@@ -0,0 +1,91 @@
+import getCSSModuleLocalIdent from 'react-dev-utils/getCSSModuleLocalIdent';
+
+// style files regexes
+const cssRegex = /\.css$/;
+const cssModuleRegex = /\.module\.css$/;
+const sassRegex = /\.(scss|sass)$/;
+const sassModuleRegex = /\.module\.(scss|sass)$/;
+
+// common function to get style loaders
+const getStyleLoaders = (cssOptions, preProcessor) => {
+  const loaders = [
+    require.resolve('style-loader'),
+    {
+      loader: require.resolve('css-loader'),
+      options: cssOptions,
+    },
+    {
+      // Options for PostCSS as we reference these options twice
+      // Adds vendor prefixing based on your specified browser support in
+      // package.json
+      loader: require.resolve('postcss-loader'),
+      options: {
+        // Necessary for external CSS imports to work
+        // https://github.com/facebook/create-react-app/issues/2677
+        ident: 'postcss',
+        postcss: {},
+        plugins: () => [
+          require('postcss-flexbugs-fixes'), // eslint-disable-line global-require
+          // eslint-disable-next-line global-require
+          require('postcss-preset-env')({
+            autoprefixer: {
+              flexbox: 'no-2009',
+            },
+            stage: 3,
+          }),
+        ],
+      },
+    },
+  ];
+  if (preProcessor) {
+    loaders.push(require.resolve(preProcessor));
+  }
+  return loaders;
+};
+
+export default [
+  // "postcss" loader applies autoprefixer to our CSS.
+  // "css" loader resolves paths in CSS and adds assets as dependencies.
+  // "style" loader turns CSS into JS modules that inject <style> tags.
+  // In production, we use a plugin to extract that CSS to a file, but
+  // in development "style" loader enables hot editing of CSS.
+  // By default we support CSS Modules with the extension .module.css
+  {
+    test: cssRegex,
+    exclude: cssModuleRegex,
+    use: getStyleLoaders({ importLoaders: 1 }),
+  },
+  // Adds support for CSS Modules (https://github.com/css-modules/css-modules)
+  // using the extension .module.css
+  {
+    test: cssModuleRegex,
+    use: getStyleLoaders({
+      importLoaders: 1,
+      modules: true,
+      getLocalIdent: getCSSModuleLocalIdent,
+    }),
+  },
+  // Opt-in support for SASS (using .scss or .sass extensions).
+  // Chains the sass-loader with the css-loader and the style-loader
+  // to immediately apply all styles to the DOM.
+  // By default we support SASS Modules with the
+  // extensions .module.scss or .module.sass
+  {
+    test: sassRegex,
```

Comment:
Wasn't this PR about to include by default only `.module.**` ? because it will probably conflict now with https://github.com/storybooks/storybook/blob/master/app/angular/src/server/framework-preset-angular.js#L35

Reply:
I updated the angular style rules filter as suggested in #4343 so it shouldn't conflict.

Though the suggested solution of `rule.test.test()` doesn't work well with [all valid options](https://webpack.js.org/configuration/module/#condition) of `test`. Working on resolving that now.

Reply:
The thing is when people are not using angular-cli (it's rare though), they will not have this filtering rules logic.

Reply:
We can have all these CRA rules , if moving from changing the `core` defaults to changing the `app/react` defaults

Reply:
I know 😔. I was hoping to find an interim solution. Maybe changing the `app/react` defaults won’t be as hard now that I’ve had a lot more exposure to the codebase haha.

Reply:
I don't think we should provide Sass support beyond `.modules.s(c|a)ss`

Even though you resolved the Angular issue, I think we're going beyond the original scope of the issue in trying to support _anybody_'s Sass usage.